src/Razor/test/Microsoft.VisualStudio.Razor.Integration.Test/IntializeTestFile.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Reflection;
+using Xunit.Sdk;
+
+namespace Microsoft.VisualStudio.Razor.Integration.Test
+{
+    public class IntializeTestFileAttribute : BeforeAfterTestAttribute
```

Comment:
I'm re-using the pattern from [Microsoft.AspNetCore.Razor.LanguageServer.Test](https://github.com/dotnet/razor-tooling/blob/main/src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/InitializeTestFileAttribute.cs) here.

I think you're probably right that this could be simplified but I'm trying to minimize the number of independent variables in this PR. I've filed https://github.com/dotnet/razor-tooling/issues/5905 to follow up on that.